lib/oc-tests/test_shareDir.py

```diff
@@ -95,6 +95,13 @@ def setup(step):
     check_users(config.oc_number_test_users)
 
     reset_rundir()
+    reset_server_log_file()
```

Comment:
Looks good in general, the fix for `scrape_log_file` is missing, but I can rebase this PR then and fix it.

Regarding your other points:
> config.oc_number_test_users : make this option None by default (you would need to enable mutliuser mode explictly in your configuration)

We have some tests, that rely on multiple users, e.g. shareFile, sharePermissions etc., because you can not share a file with yourself.
The currently use the config, although they rely on it being >= 3
https://github.com/cernbox/smashbox/blob/master/lib/oc-tests/test_shareFile.py#L89-L90
https://github.com/cernbox/smashbox/blob/master/lib/oc-tests/test_shareFile.py#L116-L117
https://github.com/cernbox/smashbox/blob/master/lib/oc-tests/test_shareFile.py#L183-L184

I guess we should fix them then and add the hardcoded number in such cases?